### ISSUE TITLE
Unable to build advancedBase branch

### ISSUE THREAD

**Issue by parkersprouse** [deepnight/gameBase#7]
Is the `advancedBase` branch considered stable enough to be usable? I can't seem to be able to build it.

I get an issue about
```bash
src/Game.hx:116: characters 3-17 : Unknown identifier : timeMultiplier
```

The standard `gameBase` is fine though!

**Comment by deepnight** [comment]
I'll update & fix the branch tomorrow, sorry about that :) Yes it should be stable and production ready.

**Comment by parkersprouse** [comment]
No worries, thank you for the response!

**Comment by qkdreyer** [comment]
This haven't been fixed yet @deepnight :(

**Comment by deepnight** [comment]
Sorry for the delay! It was fixed by 8e629ebdc63a4fc937ac3ba550489844f510b0b5
